draft-ietf-tls-esni.md

```diff
@@ -512,6 +512,11 @@ a `ClientECH` with the following values:
 - `enc`, as computed above; and
 - `payload`, as computed above.
 
+If optional configuration identifiers (see {{optional-configs}})) are used, the
+`config_id` field MAY be empty or randomly generated. Unless specified by the
+application using (D)TLS or externally configured on both sides,
+implementations MUST NOT use this mode.
```

Comment:
Suggest changing "MUST NOT use this mode" to "MUST compute config_id as specified in {{whatever-section-describes-ClientECH}}".

Reply:
Done.